src/sentry/issues/grouptype.py

```diff
@@ -174,11 +174,20 @@ class GroupType:
     notification_config: NotificationConfig = NotificationConfig()
     detector_handler: type[DetectorHandler] | None = None
     detector_validator: type[BaseGroupTypeDetectorValidator] | None = None
+    detector_config_schema: dict[str, Any] | None = None
```

Comment:
It's starting to feel like we just need something like `DetectorConfig` that encapsulates all of these. Probably something for a separate pr